modules/math/src/geometry/attributes/convert-to-non-indexed.ts

```diff
@@ -1,7 +1,12 @@
 /* eslint-disable */
 // @ts-nocheck
-
-export function convertBuffersToNonIndexed({indices, attributes}) {
+/**
+ * Converts indices of geometry.
+ *
+ * @param param0
+ * @returns no indexed geometry
+ */
+export function convertBuffersToNonIndexed({indices, attributes}): any {
```

Comment:
Argument types?

Reply:
> Argument types?

There is '// @ts-nocheck' on the top of the file, as it uses non-existent classes I prefer don't touch it.
Seems it's experimental.